tests/plan_stubs/test_wrapped_stubs.py

```diff
@@ -0,0 +1,145 @@
+    msgs = list(set_absolute(x_axis, 0.5, wait=True))
+    assert len(msgs) == 2
+    assert msgs[0] == Msg("set", x_axis, 0.5, group=ANY)
+    assert msgs[1] == Msg("wait", group=msgs[0].kwargs["group"])
+
+
+def test_set_absolute_with_group_and_wait(x_axis: SimMotor):
+    assert list(set_absolute(x_axis, 0.5, group="foo", wait=True)) == [
+        Msg("set", x_axis, 0.5, group="foo"),
+        Msg("wait", group="foo"),
+    ]
+
+
+def test_set_relative(x_axis: SimMotor):
+    assert list(set_relative(x_axis, 0.5)) == [
+        Msg("read", x_axis),
+        Msg("set", x_axis, 0.5, group=None),
+    ]
+
+
+def test_set_relative_with_group(x_axis: SimMotor):
+    assert list(set_relative(x_axis, 0.5, group="foo")) == [
+        Msg("read", x_axis),
+        Msg("set", x_axis, 0.5, group="foo"),
+    ]
+
+
+def test_set_relative_with_wait(x_axis: SimMotor):
+    msgs = list(set_relative(x_axis, 0.5, wait=True))
+    assert len(msgs) == 3
+    assert msgs[0] == Msg("read", x_axis)
+    assert msgs[1] == Msg("set", x_axis, 0.5, group=ANY)
+    assert msgs[2] == Msg("wait", group=msgs[1].kwargs["group"])
+
+
+def test_set_relative_with_group_and_wait(x_axis: SimMotor):
+    assert list(set_relative(x_axis, 0.5, group="foo", wait=True)) == [
+        Msg("read", x_axis),
+        Msg("set", x_axis, 0.5, group="foo"),
+        Msg("wait", group="foo"),
+    ]
+
+
+def test_move(x_axis: SimMotor, y_axis: SimMotor):
+    msgs = list(move({x_axis: 0.5, y_axis: 1.0}))
+    assert msgs[0] == Msg("set", x_axis, 0.5, group=ANY)
+    assert msgs[1] == Msg("set", y_axis, 1.0, group=msgs[0].kwargs["group"])
+    assert msgs[2] == Msg("wait", group=msgs[0].kwargs["group"])
+
+
+def test_move_group(x_axis: SimMotor, y_axis: SimMotor):
+    msgs = list(move({x_axis: 0.5, y_axis: 1.0}, group="foo"))
+    assert msgs[0] == Msg("set", x_axis, 0.5, group="foo")
+    assert msgs[1] == Msg("set", y_axis, 1.0, group="foo")
+    assert msgs[2] == Msg("wait", group="foo")
+
+
+def test_move_relative(x_axis: SimMotor, y_axis: SimMotor):
+    msgs = list(move_relative({x_axis: 0.5, y_axis: 1.0}))
+    assert msgs[0] == Msg("read", x_axis)
+    assert msgs[1] == Msg("set", x_axis, 0.5, group=ANY)
+    group = msgs[1].kwargs["group"]
+    assert msgs[2] == Msg("read", y_axis)
+    assert msgs[3] == Msg("set", y_axis, 1.0, group=group)
+    assert msgs[4] == Msg("wait", group=group)
+
+
+def test_move_relative_group(x_axis: SimMotor, y_axis: SimMotor):
+    msgs = list(move_relative({x_axis: 0.5, y_axis: 1.0}, group="foo"))
+    assert msgs[0] == Msg("read", x_axis)
+    assert msgs[1] == Msg("set", x_axis, 0.5, group="foo")
+    assert msgs[2] == Msg("read", y_axis)
+    assert msgs[3] == Msg("set", y_axis, 1.0, group="foo")
+    assert msgs[4] == Msg("wait", group="foo")
+
+
+def test_sleep():
+    assert list(sleep(1.5)) == [Msg("sleep", None, 1.5)]
+
+
+def test_wait():
+    # Waits for all groups
+    # move_on and wait are antonyms: what does move_on do?
```

Comment:
Should: I agree that `move_on` is bizarre. It should be documented in the `bps.wait` docstring rather than this comment here, which doesn't really add anything for the reader other than more confusion. Can we add an issue into the bluesky repo to document it properly then either link here or remove the comment?

Reply:
Tracked down the initial issue that added it, raised an upstream ticket to document better.
https://github.com/bluesky/bluesky/issues/1824